src/store/channels-list/saga.ts

```diff
@@ -43,6 +47,24 @@ export function* unreadCountUpdated(action) {
   yield put(receive(channelsList));
 }
 
+export function* loadUsers(action) {
```

Comment:
sooooooo. i *think* this is still the wrong place for this. haha. sorry. :)

the channel-list state/saga has a very specific purpose, which is to manage the main list of channels that is displayed in the context menu.

essentially there are (at least) two types of entities in global state: 
- there's the core domain objects like user, channel, message, etc.,
- and then there is higher-level state that manages specific pieces of the app or UI.

the channel-list piece of state is the second type, while the channels and users piece of state is the first. what we are doing here is loading the users for a given channel. it isn't something that relates to the list of channels, it relates to a specific channel. for that reason, it should live in the channel 'slice', and in a channels saga.

let me know if that makes sense or if you have any questions about it. i know this stuff is pretty complex for how many examples we have at this point. i'll try and work up a diagram in the next few days, but for now if you'd like me to take a pass and show you what i mean, i'd be happy to do that. otherwise i think we're pretty close. :)

Reply:
ok, no worries. I'll change the place for the user saga.